packages/binding-mqtt/test/mqtt-client-subscribe-test.ts

```diff
@@ -28,66 +27,71 @@ import { Servient, ExposedThing } from "@node-wot/core";
 import MqttBrokerServer from "../dist/mqtt-broker-server";
 import MqttClientFactory from "../dist/mqtt-client-factory";
 
-
-
-
+var osToSkip = ["windows-latest", "macos-latest"];
 
 @suite("MQTT implementation")
 class MqttClientSubscribeTest {
-
-    @test.skip "should expose via broker"(done: Function) {
-
+    @test "should expose via broker"(done: Function) {
         try {
-
             let servient = new Servient();
 
-            let brokerServer = new MqttBrokerServer("mqtt://test.mosquitto.org:1883");
+            if (osToSkip.includes(process.env.matrix_os)) {
+                done(); // TODO: should be replaced with a skip()
```

Comment:
I did not find a way yet in `mocha-typescript` to skip a test on a certain condition (while this is possible in the "regular" `mocha` library).